app/views/candidate_mailer/submit_application_email.text.erb

```diff
@@ -12,14 +12,24 @@ Your application reference is <%= @application_form.support_reference %>.
 
 You can track the progress of your <%= 'application'.pluralize(@application_form.application_choices.count) %> here:
 
-<%= candidate_interface_application_form_url %>
+<% if FeatureFlag.active?('encrypted_candidate_id') %>
+  <%= candidate_interface_application_form_url(id: @encrypted_candidate_id) %>
```

Comment:
Use `candidate_interface_sign_in_url(id: @encrypted_candidate_id)` here instead so we link to a single URL from emails. The effect is the same - if the user is already signed in they'll be redirected to `/candidate/application` anyway, and if they're not signed in they see the "expired" page.